xbmc/video/ContextMenus.cpp

```diff
@@ -426,8 +426,11 @@ bool CVideoPlay::Execute(const std::shared_ptr<CFileItem>& itemIn) const
 
 bool CVideoPlayUsing::IsVisible(const CFileItem& item) const
 {
-  if (item.HasVideoVersions())
-    return false; // display "Play version using..." if multiple versions are available.
+  if (item.HasVideoVersions() &&
+      !CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
+          CSettings::SETTING_VIDEOLIBRARY_SHOWVIDEOVERSIONSASFOLDER) &&
+      !VIDEO::IsVideoAssetFile(item))
```

Comment:
reading the setting is not necessary (think navigation from versions node or other ways to land on the version path)
It's not necessary to eliminate it on the movie/folder hybrid either (would play the default version, same as Play)

Reply:
> It's not necessary to eliminate it on the movie/folder hybrid either (would play the default version, same as Play)

Currently it is not shown for the hybrid and this should stay like this otherwise we have both „play version using“ and „play using“.

Reply:
I runtime-tested one more time. If I remove the line reading the setting, like @CrystalP suggests, behaviour is not as intended:

Setting "show videos with multiple versions" OFF
* ctx menu for a hybrid 
** has play version using -> okay
** has no play using -> okay
* ctx menu of a single version listed in Movies/Versions/edition-name
** has no play version using -> okay
** has play using -> okay

Setting "show videos with multiple versions" ON
* ctx menu for a hybrid 
** has no play version using -> okay
** has no play using -> NOT okay
* ctx menu of a single version listed in hybrid-name
** has no play version using -> okay
** has play using -> okay
* ctx menu of a single version listed in Movies/Versions/edition-name
** has no play version using -> okay
** has play using -> okay

I think implementation is correct as it is in this PR. Especially it seems right the the condition is exactly the opposite of the visible condition used for `CVideoPlayVersionUsing::IsVisible` and the goal is to shown the one or the other, not both.

Reply:
> > It's not necessary to eliminate it on the movie/folder hybrid either (would play the default version, same as Play)
> 
> Currently it is not shown for the hybrid and this should stay like this otherwise we have both „play version using“ and „play using“.

your previous PR eliminated "Play version using...", no? That's where I commented on the gap addressed here

Reply:
> your previous PR eliminated "Play version using..." on the movie/folder, no? That's where I commented on the gap addressed here.

Yes, exactly.

And this PR is supposed the issue you where mentioning here: https://github.com/xbmc/xbmc/pull/24670#issuecomment-1935448852

> it created a dissymetry between setting on/off.

Not sure I understand.

Does this PR fix the problem? Does this PR introduce any regression?

Reply:
sorry for the brainfart, I should have runtime tested first. Yes this is ok. no regression but I found a new (existing) gap.